package.json

```diff
@@ -9,6 +9,7 @@
     "url": "https://github.com/percy/cli"
   },
   "scripts": {
+    "build_cjs": "BABEL_ENV=dev babel packages -d build",
```

Comment:
can this be linked with lerna command?

lerna is mono repo package manager, check all the commands below for reference.

Reply:
[optional] hook with existing `build` command 

so following without `ENV` builds normally
```
yarn build
```

and following builds with the hacks present in `build.sh` ( we'll have `if else` in codebase which is fine and testable)

```
BABEL_ENV=executable yarn build
```

Reply:
i don’t think this is possible `babel packages -d build ` command is converting everything present inside packages folder of cli to cjs format supported by `pkg`.

Reply:
lets tackle in followup PR.